solutions/client-project-setup/namespaces/project-id-tier3.yaml

```diff
@@ -98,6 +98,24 @@ spec:
   role: organizations/org-id/roles/tier3.dnsrecord.admin # kpt-set: organizations/${org-id}/roles/tier3.dnsrecord.admin
   member: "serviceAccount:tier3-sa@project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:tier3-sa@${project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP Project IAM Admin to GCP SA on Client Host Project
```

Comment:
to be validated

Reply:
removed in last commit as I have validated that it was an old requirement that no longer exist